src/components/network-area-diagram-viewer/network-area-diagram-viewer.ts

```diff
@@ -1024,6 +1108,9 @@ export class NetworkAreaDiagramViewer {
         traversingBusEdgesAngles.push(
             traversingBusEdgesAngles[0] + 2 * Math.PI
         );
+        traversingBusEdgesAngles.sort(function (a, b) {
```

Comment:
I don't get why this is needed, the angles are all computed with atan2, so are between -pi and pi. The last angle added is therefore between pi and 3*pi, the second sorting shouldn't change anything. What am I missing?

Reply:
This second sorting was added to correctly handle the redrawing after some node moving. Is seems that changing the addition of the last angle to the angle list would make this second sorting useless (so it has been removed).